index.html

```diff
@@ -1,182 +1,255 @@
+      </ul>
+
+      <button class="see" type="button">See Project</button>
+
+      <div class="project">
+        <img
+          class="sample"
+          src="./images/sample-image.png"
+          alt="Project Sample Image"
+        />
         <h3 class="title">Multi-Post Stories</h3>
-        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-            dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        <p class="text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
         </p>
-    </div>
-    
-    <ul class="lang">
+      </div>
+
+      <ul class="lang">
         <li class="css">css</li>
         <li class="liner">|</li>
         <li class="html">html</li>
         <li class="liner-2">|</li>
         <li class="boot">bootstrap</li>
         <li class="liner-3">|</li>
         <li class="ruby">Ruby</li>
-    </ul>
-    
-    <button class="see" type="button">See Project</button>
+      </ul>
 
- </section>
+      <button class="see" type="button">See Project</button>
+    </section>
 
-   
-</body>
-</html>
\ No newline at end of file
+    <section>
+      <div class="contact-form-container">
+        <h2 class="contacts-title">CONTACT US</h2>
+        <p class="contact-texts">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+
+      <div class="forms">
+        <form action="https://formspree.io/f/xdobrapq" method="post">
+         
+              <input
+                type="text"
+                placeholder="Text Input"
+                required
+                maxlength="30"
+              />
+            
+              <input
+              class="text-email"
+                type="email"
+                placeholder="jeremy@gmail.com"
+                required
+                maxlength="30"
+              />
+           
+              <textarea
+                name="message"
+                id="msg"
+                required
+                maxlength="500"
+                placeholder="Enter text here"
+              ></textarea>
+         
+          <button type="submit" class="submit-btn">See Project</button>
+        </form>
+      </div>
+      <hr />
+      <div class="contacts">
+        <img src="./images/Vector(1).png" alt="Github" />
+        <img src="./images/Vector(2).png" alt="Chill" />
+        <img src="./images/Vector(3).png" alt="Twitter icon" />
+        <img class="link" src="./images/Group.png" alt="linkedin icon" />
+        <img src="./images/Vector(4).png" alt="M ICON" />
+      </div>
```

Comment:
Great work team 🤝

- [ ] [Optional] You may try to make your social icons very identical to the design provided in Figma.
- Your design
![your social](https://user-images.githubusercontent.com/63917725/177420953-187f736f-1061-40b1-abac-7c1efc1259fa.PNG)

- Figma design
![figma social](https://user-images.githubusercontent.com/63917725/177420998-c940516b-6409-4dab-9d5d-1d111a10168c.PNG)